eth/tracers/js/tracer_test.go

```diff
@@ -130,7 +130,7 @@ func TestTracer(t *testing.T) {
 			want: `["PUSH1","PUSH1","STOP"]`,
 		}, { // tests intrinsic gas
 			code: "{depths: [], step: function() {}, fault: function() {}, result: function(ctx) { return ctx.gasPrice+'.'+ctx.gasUsed+'.'+ctx.intrinsicGas; }}",
-			want: `"100000.6.21000"`,
+			want: `"100000.30994.21000"`,
```

Comment:
So the execution of the three opcodes `PUSH1, PUSH1, STOP` burns `30994` gas? How is that possible?

Reply:
Good point haha, there was an issue in `runTrace`. Fixed.